app/dashboard/embed.py

```diff
@@ -217,14 +217,7 @@ def embed(request):
         i = 0
         for bounty in bounties:
             i += 1
-            value_eth = str(round(bounty.value_in_eth/10**18, 2)) + "ETH" if bounty.value_in_eth else ""
-            value_in_usdt = str(round(bounty.value_in_usdt, 2)) + "USD" if bounty.value_in_usdt else ""
-            eth_value_in_usdt = str(round(get_eth_to_usdt(), 2)) + "/ETH"
-            value_native = "{} {}".format(round(bounty.value_true, 2), bounty.token_name)
-
-            value = "{}, (${} @ {})".format(value_eth, value_in_usdt, eth_value_in_usdt)
-            if not value_eth:
-                value = value_native
+            value = "{} {}, (${} @ {})".format(round(bounty.value_true, 2), bounty.token_name, round(bounty.value_in_usdt, 2), round(convert_token_to_usdt(bounty.token_name), 2))
```

Comment:
I struggled to keep this as it was but it just seemed overly complicated to me. I think all the logic above boils down to the one line but please spot me and hold me honest! :-S thanks!